requirements/openvino.txt

```diff
@@ -2,7 +2,7 @@
 # OpenVINO Requirements.                                                      #
 nncf==2.5.0
 onnx==1.13.0
-openvino-model-api==0.1.4
+openvino-model-api @ git+https://github.com/openvinotoolkit/model_api/@5de9079b5742ed34c8ccb67c3bacb38773dffd8f#egg=openvino-model-api&subdirectory=model_api/python
```

Comment:
This will prevent OTX PyPI packaging. When do you expect this MAPI commit to be published via PyPI version?

Reply:
I expect this after merging https://github.com/openvinotoolkit/model_api/pull/118 to MAPI